dotenv/__init__.py

```diff
@@ -1,3 +1,5 @@
+__version__ = '0.6.4'
+
 from .cli import get_cli_string
```

Comment:
Since the setup.py's entrypoint is cli, I don't think you need to impoot anything from .cli here